data/AlchemicalArtist.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Chris Denton",
+  "displayStatsPublic": false,
+  "type": "personal",
+  "bio": "artist, musician, human",
+  "avatar": "https://GitHub.com/alchemicalArtist.png",
```

Comment:
Your username is not the exact same case as your GitHub username, this will work now but might not in the future, it is best to keep it exactly the same

Please do the same with the file name also, have it match your GitHub username exactly